test/async.js

```diff
@@ -0,0 +1,80 @@
+"use strict"
+
+/* globals Promise:true */
+
+var assert = require("../index")
```

Comment:
Nit: load and use `../async` instead of `../index`, and edit your stuff accordingly. (You won't need to filter the imports.)